warema_wms/shade.py

```diff
@@ -105,6 +141,24 @@ def _verify_set_cmd_sent(self, target_position):
             time.sleep(self.time_between_cmds)
         return False
 
+    def _verify_tilt_cmd_sent(self, target_tilt_position):
+        time.sleep(self.time_between_cmds)
+        for _ in range(self.num_retries):
+            self.update_shade_state()
+            if self.is_moving or self.tilt_position == target_tilt_position:
```

Comment:
I don't know if `is_moving` is `True` when only the tilt is being changed. Probably something that should be verified.

Reply:
have to check this an come back

Reply:
technically, the angle is realized using the same pull rope as moving up and down (as mentioned elsewhere, move up = fully open, down = fully closed), i.e. in principle the status should also be true during tilt adjustment , but only for a short time (full open to fully closed needs 1 or maybe 2 seconds), no idea if that's not too short?

Reply:
i did several trials yesterday, but was not able to receive a true state during tilt adjustment. 
i was not able to determine, if its caused by the by the short time it need to change from one state to another, (and i missed everytime the right time slot for the request) or the state is actually false during tilt movement.

the update command was send always via console, the tilt command via Webui of the webcontrol box. 

do you have any ideas how i could check this, pls give advice.

correction to my last post, it takes almost exactly 1 second to change tilt from full open to full closed (from the moment slats start moving)

Reply:
I would leave it as it is given that your argument that the same pull rope is used is very valid.